cypress/e2e/authoring/carousel-question.cy.js

```diff
@@ -6,7 +6,7 @@ const carouselAuthoringPage = new CarouselAuthoringPage;
 
 context("Test Authoring Preview", () => {
   before(() => {
-    cy.visit("https://lara2-staging.concordqa.org/");
+    cy.visit(Cypress.config().baseUrl);
```

Comment:
@saravanan-udayakumar , I'm pretty sure this can be changed to `cy.visit("");` everywhere it's used.
Cypress should pickup the baseUrl from the config properties and it shouldn't have to be explicitly asked to do so.

Reply:
@tejal-shah , updated the changes to all the tests.